### PR TITLE
update documentation

### LINKED ISSUE #29: Add cf_role to the attributes of the 'id' dimension
To follow CF Conventions, discrete datasets with a `station` dimension (no standardized name, so `id` works too) should have a `cf_role` attribute with a value `"timeseries_id"`. This is what allows the right dimension to be easily identified, similarly to the `axis` attribute for `lon/lat`.

http://cfconventions.org/cf-conventions/cf-conventions.html#time-series-data

> It is strongly recommended that there should be a station variable (which may be of any type) with the attribute cf_role=”timeseries_id”, whose values uniquely identify the stations.

### PR DESCRIPTION
**Pull request by sebastienlanglois**
### Pull Request Checklist:
- [X] This PR addresses an already opened issue (for bug fixes / features)
  - This PR fixes #29
- [X] (If applicable) Documentation has been added / updated (for bug fixes / features).
- [ ] (If applicable) Tests have been added. -> Will do in an upcoming PR
- [X] CHANGES.rst has been updated (with summary of main changes).
  - [X] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.

### What kind of change does this PR introduce?

* Updating documentation

### Does this PR introduce a breaking change?
Yes, we changed the output format for climatological datasets in order to minimize the amount of dimensions. This won't break current documentation in xhydro using xdatasets

### CODE REVIEW

**Comment by sebastienlanglois** [comment]
@Zeitsperre
I'm using tqdm the track computing done in xdatasets. However, when building the docs, I see this strange behaviour. Have you seen this before ? I haven't changed anything that I can think of that should impact tqdm. Any clues ?

image

**Comment by sebastienlanglois** [comment]
@RondeauG @Zeitsperre 
Cette PR met à jour la documentation, permet d'extraire des datasets GIS (ex: polygons) et fait un peu de ménage dans les workflows. Comme nous avons besoin rapidement de ces fonctionnalités, si vous êtes d'accord d'approuver ce PR sans les tests, je vais les ajouter dans un PR subséquent.

**Review by RondeauG — APPROVED** [review]
LGTM!

**Comment by RondeauG** [comment]
Comme il n'y a pas encore de tests dans `xdatasets`, je ne vois pas d'enjeu à les skipper pour le moment. Tu pourras les implémenter à ton rythme.

File: xdatasets/core.py [hunk 1]
```diff
@@ -259,12 +264,20 @@ def load_query(
...
+            if type(dsets[0]) == xr.Dataset:
+                # if more than one dataset, then we add source as a dimension
+                # so we can merge two or more datasets together
+                if len(dsets) > 1:
+                    for idx, dset in enumerate(dsets):
+                        for var in dset.keys():
+                            dset[var] = dset[var].expand_dims("source", axis=-1)
+                        dsets[idx] = dset
+                ds = xr.merge(dsets)
+                ds = ds
```
[the remark this answers is not in the document]
Comment (sebastienlanglois) [reply xdatasets/core.py:276]: Indeed!

File: docs/notebooks/getting_started.ipynb [hunk 2]
```diff
@@ -968,53 +395,51 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The two plots depicted above show the 2m temperature for each pixel that intersects with the polygon from Station `023003` and the corresponding weights to be applied to each pixel. In the lower plot, it is apparent that the majority of the polygon is situated in the upper-left pixel, which results in that pixel having a weight of approximately 91%. It is evident that the two lower pixels have very minimal intersection with the polygon, which results in their respective weights being nearly zero (hover on the plot to verify the weights).\n",
+    "The two plots depicted above show the 2m temperature for each pixel that intersects with the polygon from Station `042102` and the corresponding weights to be applied to each pixel. In the lower plot, it is apparent that the majority of the polygon is situated in the central pixels, which results in those pixelx having a weight of approximately 80%. It is evident that the two lower and the upper pixels have much less intersection with the polygon, which results in their respective weights being smaller (hover on the plot to verify the weights).\n",
```
[the remark this answers is not in the document]
Comment (sebastienlanglois) [reply docs/notebooks/getting_started.ipynb]: Solved

File: xdatasets/core.py [hunk 3]
```diff
@@ -259,12 +264,20 @@ def load_query(
...
+            elif len(dsets) == 1:
```
[the remark this answers is not in the document]
Comment (sebastienlanglois) [reply xdatasets/core.py:278]: I've added a ds at the top of this section so that it's always defined

**Comment by sebastienlanglois** [comment]
> @Zeitsperre I'm using tqdm the track computing done in xdatasets. However, when building the docs, I see this strange behaviour. Have you seen this before ? I haven't changed anything that I can think of that should impact tqdm. Any clues ?
> 

Solved it, it was caused by : https://github.com/jupyter/nbconvert/issues/2092

**Pull request merged**